src/chapters/jdbc-auth.xml

```diff
@@ -97,9 +97,51 @@
                         depends on the version of Java you have installed. </para>
                 </listitem>
             </varlistentry>
+            <varlistentry>
+                <term><filename>sqlserver/</filename></term>
+                <listitem>
+                    <para>Contains the SQL Server authentication extension,
+                        <filename>guacamole-auth-jdbc-sqlserver-0.9.13-incubating.jar</filename>,
+                        along with a <filename>schema/</filename> directory contains SQL Server-specific
+                        scripts requires to set up the database.  The JAR extension file will need to be
+                        placed within the <filename>GUACAMOLE_HOME/extensions</filename> folder, while the
+                        SQL Server JDBC driver must be placed within the <filename>GUACAMOLE_HOME/lib</filename>
+                        directory.</para>
+                    <para><emphasis>The SQL Server JDBC driver is not included with the extension.</emphasis>  You
+                        must obtain the JDBC driver <filename>.jar</filename> yourself and place it in the directory.
+                        Furthermore, the SQL Server authentication extension supports a number of TDS-compatible
+                        drivers, so you must make sure the one you choose is supported by the extension, that the
+                        extension is configured properly, and that the <filename>.jar</filename> is in the correct
+                        directory.  Microsoft's JDBC driver can be downloaded from Microsoft's <link
+                        xlink:href="https://docs.microsoft.com/en-us/sql/connect/sql-connection-libraries#anchor-20-drivers-relational-access">
+                        SQL Connection Libraries</link> page.</para>
+                    <para>In addition to the various parameters mentioned below, the SQL Server driver has a
+                        unique parameter available to control the driver compatibility of the JDBC module:
+                        <property>sqlserver-driver</property>.  This parameter allows you to choose the compatibility
+                        mode of the module with various TDS-comptabile drivers such that it can be used with different
+                        versions of SQL Server and even non-Microsoft SQL Server databases.  The following options are available
+                        for the <property>sqlserver-driver</property> property:
+                        <variablelist>
```

Comment:
Not sure what's going on, but the list of values isn't actually appearing in the HTML output of the manual build:

![section-contents-missing](https://user-images.githubusercontent.com/4632905/34449823-b13b99d8-ecb1-11e7-8054-a7f4c6f6e256.png)

I'm guessing that it might have to do with the `<variablelist>` being inside the `<para>`, but not sure.

Reply:
Looks like I was missing the `<listitem>` tags.  Also changed it from `<variablelist>` to `<itemizedlist>` to get the desired bullet effect.